tools/jarcat/zip/writer_test.go

```diff
@@ -24,10 +24,64 @@ func TestAddZipFile(t *testing.T) {
 	assertExpected(t, "add_zip_file_test.zip", 0)
 }
 
+// getZipContents is a helper which returns a map of filename -> contents
+func getZipContents(zipfilePath string) (map[string][]byte, error) {
```

Comment:
please let me know if there's a more straightforward way

Reply:
Not that I can think of right now.